tests/test_error_handlings.c

```diff
@@ -31,11 +31,13 @@ void test_incorrect_password(void **unused) {
     SF_STATUS status = snowflake_connect(sf);
     assert_int_not_equal(status, SF_STATUS_SUCCESS); // must fail
 
+/*
```

Comment:
Is this a planned change?
If so, maybe remove it instead of commenting.

Reply:
comment added.